packages/grafana-runtime/src/measurement/perf.ts

```diff
@@ -0,0 +1,28 @@
+let lastUpdate = Date.now();
+
+/**
+ * This object indicats how overloaded the main thread is
+ */
+export const perf = {
+  budget: 1,
+  threshold: 1.05, // trial and error appears about right
+  ok: true,
+  last: lastUpdate,
+};
+
+// Expose this as a global object so it can be changed locally
+// NOTE: when we are confident this is the right budget, this should be removed
+(window as any).grafanaStreamingPerf = perf;
```

Comment:
@leeoniya better?

Reply:
lemme test if the loop is still 1ms / 10s...

Reply:
:+1: